gcloud/bigtable/client.py

```diff
@@ -375,22 +376,33 @@ def __exit__(self, exc_type, exc_val, exc_t):
         """Stops the client as a context manager."""
         self.stop()
 
-    def instance(self, instance_id, display_name=None):
+    def instance(self, instance_id, location,
+                 display_name=None, serve_nodes=DEFAULT_SERVE_NODES):
         """Factory to create a instance associated with this client.
 
         :type instance_id: str
         :param instance_id: The ID of the instance.
 
+        :type location: string
+        :param location: location name, in form
+                         ``projects/<project>/locations/<location>``; used to
```

Comment:
Doesn't the client already know the project? Could the bare location be the only argument here?